src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java

```diff
@@ -79,9 +79,7 @@
     private YesOrNo lastNameChangedWhenMarried;
 
     @CCD(
-        label = "Have they changed their name since they got married?",
-        hint = "Is their current name different to their married name or the name shown on their "
-            + "marriage certificate?"
+        label = "Have they changed their name since they got married?"
```

Comment:
for civil partnership married won't be applicable ? could you plz check with Michael on this ?